operator/pkg/central/extensions/reconcile_scanner_db_password_test.go

```diff
@@ -37,50 +37,50 @@ func TestReconcileScannerDBPassword(t *testing.T) {
 
 	cases := map[string]secretReconciliationTestCase{
 		"When no db password secret exists and scanner is disabled, no secrets should be created": {
-			Spec:                   basicSpecWithScanner(false),
+			Spec:                   basicSpecWithScanner(false, false),
```

Comment:
When looking at how `basicSpecWithScanner` is used, I wonder if we really need it to be configurable at the level of specific scanner versions (v2 vs. v4) -- wouldn't it suffice to keep one boolean and apply it to both scanner configurations?

Reply:
not a blocker, feel free to resolve.

Reply:
Overlooked this, but I think it's not sufficient because test case asserts are on that level, so we'd need to modify a lot of existing cases. That's why I rather did this and added new test cases.